### PR TITLE
notebook 5 comments

### PR DESCRIPTION
**Pull request by SophieRowland**
This is my last one for the day. Let me know when you'd like me to review the changes.

### CODE REVIEW

File: docs/src/05-mgx-taxonomic-profiles.jmd [hunk 1]
```diff
@@ -4,6 +4,7 @@ Taxonomic profiles come from [MetaPhlAn2](https://bitbucket.org/biobakery/metaph
 Each sample is run separately, and needs to be joined in a single table.
 I'll use the function [`load_taxonomic_profiles`]@ref
 
+**What does `@ref` mean?**
```
Comment (kescobo) [inline docs/src/05-mgx-taxonomic-profiles.jmd]: It's a thing that links to docstrings when using a package called `Documenter` which I was trying to use before. It's having problems that I don't want to deal with atm. Thanks for spotting.

File: docs/src/05-mgx-taxonomic-profiles.jmd [hunk 2]
```diff
@@ -79,12 +87,14 @@ Using `pairwise` from `Distances.jl` and `MDS` from `MultivariateStats.jl`,
 we perform classical multidimensional scaling (which is the same thing
 as principle coordinates analysis).
 
+**Principle coordinate or coordinates?**
```
Comment (kescobo) [inline docs/src/05-mgx-taxonomic-profiles.jmd]: According to the internet, it's coordinate**s**, but should be princip**al**.

**Comment by kescobo** [comment]
@SophieRowland Changes here ready for your review

File: docs/src/05-mgx-taxonomic-profiles.jmd [hunk 3]
```diff
@@ -274,31 +273,40 @@ savefig(joinpath(figures, "taxonomic-profiles-kids-shannon.svg"));
 Let's look at the phyla too:
 
 ```julia
-
 pkids = view(pabt, sites=firstkids(resolve_sampleID.(samplenames(abt))))
 
 kids_bact = vec(collect(occurrences(view(pkids, species=occursin.("Bact", speciesnames(pkids))))))
 kids_firm = vec(collect(occurrences(view(pkids, species=occursin.("Firm", speciesnames(pkids))))))
 kids_act = vec(collect(occurrences(view(pkids, species=occursin.("Actino", speciesnames(pkids))))))
 kids_proteo = vec(collect(occurrences(view(pkids, species=occursin.("Proteo", speciesnames(pkids))))))
```
Comment (SophieRowland) [inline docs/src/05-mgx-taxonomic-profiles.jmd]: Screen Shot 2019-07-08 at 12 02 13 PM
This code prints this output. Is it supposed to do this? If so, unclear what it means.

File: docs/src/05-mgx-taxonomic-profiles.jmd [hunk 4]
```diff
@@ -368,33 +376,30 @@ We can also annotate this plot with the age of the kids
 ```julia
 plot(
     abundanceplot(kids, srt = kids_hcl.order, title="Kids, top 10 species",
-        xticks=false, color=color4', legend=false),
+        xticks=false, color=color4', legend=false, linecolor=:match, linestyle=:dot),
     heatmap(collect(Float64[ismissing(x) ? 0 : x / 365 for x in focusmeta[:correctedAgeDays][kids_hcl.order]]'),
         xticks=false, yticks=false, color=:PuBu, colorbar=false),
     layout = grid(2,1,heights=[0.9, 0.1])
     )
 ```
```
Comment (SophieRowland) [inline docs/src/05-mgx-taxonomic-profiles.jmd:386]: Screen Shot 2019-07-08 at 12 10 02 PM
This plot doesn't have a legend for the age annotation.

**Comment by SophieRowland** [comment]
Plots all look good! Just a couple comments above to address. (No commits)

**Pull request merged**